### PR TITLE
RET-5209 - Add claimant to excel sheet - Deployed to Demo

### PR DESCRIPTION
**Pull request by marcjudge**
### Jira link (if applicable)
https://tools.hmcts.net/jira/browse/RET-5209

### Change description ###
Add claimant to excel sheet

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -490,7 +490,7 @@ bootWithCCD {
 
 }
 
-pmdMain.maxFailures = 306
-pmdTest.maxFailures = 207
+pmdMain.maxFailures = 264
+pmdTest.maxFailures = 184
```
Comment (hsjhita) [inline build.gradle:494]: Great work on bringing this down!

**Review by hsjhita — APPROVED** [review]

**Review by hsjhita — DISMISSED** [review]

**Review by fergusoshea — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelCreationService.java [hunk 2]
```diff
@@ -19,43 +20,67 @@
...
+            CellStyle styleForLockingLead = getStyleForLocking(workbook, true);
+            CellStyle styleForClaimant = getStyleForClaimant(workbook);
+
+            initializeHeaders(sheet, styleForLocking);
+            initializeData(sheet, multipleCollection,
+                    subMultipleCollection, leadCaseString, styleForUnLocking, styleForLocking, styleForLockingLead,
+                    styleForClaimant, userToken, caseTypeId);
+
+            adjustColumnSize(sheet);
+            createHiddenSheet(workbook, hiddenSheet, subMultipleCollection);
```
Comment (fergusoshea) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelCreationService.java:73]: This should have styleForLocking passed in and workbook is no longer needed.

File: src/cftlib/resources/compose/docker-compose.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ services:
       IDAM_TESTING_SUPPORT_ENABLED: "true"
       LOGGING_LEVEL_UK_GOV_HMCTS_IDAM: WARN
       IDAM_USER_BASE_URI: http://localhost:5062
-      IDAM_S2S_BASE_URI: http://host.docker.internal:8489
+      IDAM_S2S_BASE_URI: http://172.19.164.106:8489
```
Comment (fergusoshea) [inline src/cftlib/resources/compose/docker-compose.yml:25]: Probably want to exclude this

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelDocManagementService.java [hunk 4]
```diff
@@ -49,15 +49,16 @@ public class ExcelDocManagementService {
 
     public void uploadExcelDocument(String userToken, MultipleDetails multipleDetails, byte[] excelBytes) {
         MultipleData multipleData = multipleDetails.getCaseData();
-        log.info("Multiple Name is: " + multipleData.getMultipleName() + "for multiple reference: "
-                + multipleData.getMultipleReference());
+        log.info("Multiple Name is: {}for multiple reference: {}",
```
Comment (fergusoshea) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelDocManagementService.java:52]: I know the spacing isn't right in the original, but can we separate "{}for" -> "{} for" please?

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelDocManagementService.java [hunk 5]
```diff
@@ -91,16 +92,16 @@ public void generateAndUploadExcel(List<?> multipleCollection, String userToken,
     public void writeAndUploadExcelDocument(List<?> multipleCollection, String userToken,
                                             MultipleDetails multipleDetails, List<String> subMultipleCollection) {
         MultipleData multipleData = multipleDetails.getCaseData();
-        log.info("MultipleName is: " + multipleData.getMultipleName() + "for multiple reference: "
-                + multipleData.getMultipleReference());
+        log.info("MultipleName is: {}for multiple reference: {}",
```
Comment (fergusoshea) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/excel/ExcelDocManagementService.java:95]: Same here please: "{} for"

**Review by ss-evoco — DISMISSED** [review]

**Review by ss-evoco — APPROVED** [review]

**Review by fergusoshea — APPROVED** [review]

**Pull request merged**